index.html

```diff
@@ -4051,6 +4052,93 @@ <h4>Plain HTML or Polyfill DOM Result of the MathML Quadratic Formula</h4>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="meter">
+			<rdef>meter</rdef>
+			<div class="role-description">
+				<p>An <a>element</a> that represents a scalar measurement within a known range, or a fractional value. See related <rref>progress</rref>.</p>
+				<p>Authors MUST NOT use the <code>meter</code> role to represent a value whose minimum and maximum values are unknown and MUST use <pref>aria-valuemin</pref> and <pref>aria-valuemax</pref> to make these values available to assistive technologies.</p>
```

Comment:
Can we split this into 2 sentences?

Reply:
Done in the latest version. Thanks for the review!